packages/next/src/server/stream-utils/node-web-streams-helper.test.ts

```diff
@@ -188,7 +190,77 @@ describe('node-web-stream-helpers', () => {
+          start(controller) {
+            controller.enqueue(encoder.encode('<data>inlined data</data>'))
+            controller.close()
+          },
+        })
+      const defaultGetServerInsertedHTMLFactory = () => {
+        const serverInsertedHTMLStream = new ReadableStream({
+          start(controller) {
+            controller.enqueue('<server-html>Server HTML</server-html>')
+            controller.close()
+          },
+        })
+        const reader = serverInsertedHTMLStream.getReader()
+        return async () => {
+          const { value } = await reader.read()
+          return value
+        }
+      }
+      const defaultOptionsFactory = () => {
+        // values hardcoded based on usage of `continueFizzStream` in `server/render.tsx`
+        return {
+          isStaticGeneration: true, // always true
+          serverInsertedHTMLToHead: false, // always false
+          validateRootLayout: undefined, // always undefined
+          inlinedDataStream: defaultInlinedDataStreamFactory(),
+          getServerInsertedHTML: defaultGetServerInsertedHTMLFactory(),
+          suffix: defaultSuffix,
+        }
+      }
+      const defaultHTMLData = `<html><head><title>My Website</title></head><body><div><h1>My Website</h1></div></body></html>;`
+      const defaultHTMLDataEncoded = encoder.encode(defaultHTMLData)
+      const defaultReactReadableStreamFactory = (byteCount: number = 4) =>
+        new ReadableStream({
+          async pull(controller) {
+            for (let i = 0; i < defaultHTMLDataEncoded.length; i += byteCount) {
+              // 4 bytes at a time
+              controller.enqueue(defaultHTMLDataEncoded.slice(i, byteCount))
+              await Promise.resolve() // await one microtask
+            }
+            controller.close()
+          },
+        })
+      it.only('should continue fizz stream operation using default arguments', async () => {
+        const input: ReactReadableStream = defaultReactReadableStreamFactory(8)
+        // TODO: Somehow spy on `input.allReady` and assert it gets awaited
+        const output = await continueFizzStream(input, defaultOptionsFactory())
+        const expected = encoder.encode(
+          '<server-html>Server HTML</server-html><html><head><title>My Website</title></head><body><div><h1>My Website</h1></div>;<data>inlined data</data><suffix>suffix</suffix></body></html>'
+        )
+        const actual = new Uint8Array(expected.length)
+        let i = 0
+        const reader = output.getReader()
+        let { done, value } = await reader.read()
+        const decoder = new TextDecoder()
+        console.log({ done, value, decoded: decoder.decode(value) })
+        while (!done && value) {
+          actual.set(value, i)
+          i += value.length
+          ;({ done, value } = await reader.read())
+          console.log({ done, value, decoded: decoder.decode(value) })
+        }
+        expect(actual).toStrictEqual(expected)
+      })
```

Comment:
@gnoff I tried changing this test to use an input that is more realistic (based off of your review in https://github.com/vercel/next.js/pull/63011), but now I'm running into something strange. 

When I run this code, I get the following output:

```
> jest --runInBand "packages/next/src/server/stream-utils/node-web-streams-helper.test.ts"

  console.log
    {
      done: false,
      value: Uint8Array(38) [
         60, 115, 101, 114, 118, 101, 114,  45,
        104, 116, 109, 108,  62,  83, 101, 114,
        118, 101, 114,  32,  72,  84,  77,  76,
         60,  47, 115, 101, 114, 118, 101, 114,
         45, 104, 116, 109, 108,  62
      ],
      decoded: '<server-html>Server HTML</server-html>'
    }

      at Object.log (../packages/next/src/server/stream-utils/node-web-streams-helper.test.ts:253:17)

  console.log
    {
      done: false,
      value: Uint8Array(8) [
         60, 104, 116, 109,
        108,  62,  60, 104
      ],
      decoded: '<html><h'
    }

      at Object.log (../packages/next/src/server/stream-utils/node-web-streams-helper.test.ts:258:19)

  console.log
    {
      done: false,
      value: Uint8Array(25) [
         60, 100,  97, 116,  97,  62, 105,
        110, 108, 105, 110, 101, 100,  32,
        100,  97, 116,  97,  60,  47, 100,
         97, 116,  97,  62
      ],
      decoded: '<data>inlined data</data>'
    }

      at Object.log (../packages/next/src/server/stream-utils/node-web-streams-helper.test.ts:258:19)

  console.log
    {
      done: false,
      value: Uint8Array(23) [
         60, 115, 117, 102, 102, 105,
        120,  62, 115, 117, 102, 102,
        105, 120,  60,  47, 115, 117,
        102, 102, 105, 120,  62
      ],
      decoded: '<suffix>suffix</suffix>'
    }

      at Object.log (../packages/next/src/server/stream-utils/node-web-streams-helper.test.ts:258:19)

  console.log
    {
      done: false,
      value: Uint8Array(14) [
         60,  47,  98, 111, 100,
        121,  62,  60,  47, 104,
        116, 109, 108,  62
      ],
      decoded: '</body></html>'
    }

      at Object.log (../packages/next/src/server/stream-utils/node-web-streams-helper.test.ts:258:19)

  console.log
    { done: true, value: undefined, decoded: '' }

      at Object.log (../packages/next/src/server/stream-utils/node-web-streams-helper.test.ts:258:19)
```

It seems like only the first 8 bytes of the `defaultHTMLData` is getting passed through? What am I doing wrong?

Reply:
lmfao I'm just missing a `i+` in the slice operation. whoops.